docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,144 @@
+<script>
+  const unmountToolbar = window.SentryToolbar.init({ … });
+
+  // sometime later...
+  unmountToolbar();
+</script>
+```
+
+## Implement Feature Flag Adapter
+
+In order to integrate your Feature Flagging platform with the Dev Toolbar you will need an Adapter that can read flag data from your provider, and also store and retrieve a list of overrides to apply to your local browser session.
+
+We plan to create adapters for OpenFeature and LaunchDarkly, coming soon!
+
+The interface you need to implement for an Adapter is:
+```javascript
+type FlagValue = boolean | string | number | undefined;
+type FlagMap = Record<string, FlagValue>;
+interface FeatureFlagAdapter {
+  /**
+   * All known flag names and their evaluated values.
+   */
+  getFlagMap: () => Promise<FlagMap>;
+
+  /**
+   * Any overridden or manually set flags and values.
+   */
+  getOverrides: () => Promise<FlagMap>;
+
+  /**
+   * Manually set a flag to be a specific value, overriding the evaluated value.
+   */
+  setOverride: (name: string, override: FlagValue) => void;
+
+  /**
+   * A callback to clear all overrides from this browser.
+   */
+  clearOverrides: () => void;
+
+  /**
+   * Deeplink into your external feature-flag provider and find out more about
+   * this specific flag.
+   */
+    urlTemplate?: undefined | ((name: string) => string | URL | undefined);
+    }
+```
+
+[MockFeatureFlagIntegration.tsx](https://github.com/getsentry/sentry-toolbar/blob/301c31d561a81e3fd8ffc9532aad3a60af685916/src/env/demo/MockFeatureFlagIntegration.tsx) is an example adapter to use as a reference.
+
+## Init Configuration Fields
+
+The following fields can be passed into the `.init()` function.
+
+For example, a minimal configuration looks like:
+```javascript
+window.SentryToolbar.init({
+  organizationSlug: ‘acme’,
+  projectIdOrSlug: ‘website’,
+  environment: ‘production’
+});
+```
+
+| Field Name | Type | Description | Default Value |
+| ----- | ----- | ----- | ----- |
+| `organizationSlug` | `string` | The organization that users should login to. For example \`acme\` | *Required Value* |
+| `projectIdOrSlug` | `string \| number` | The project for which this website/webapp is associated. | *Required Value* |
+| `environment (optional)` | `string \| string[] \| undefined` | The environment of this deployment. Used to narrow search results in the Toolbar UI. Set to \`undefined\` or \`””\` or \`\[\]\` if you want to see results from all environments. | `undefined` |
+| `placement (optional)` | `‘right-edge’ \| ‘bottom-right-corner’` | Where to render the toolbar on the screen. | `”right-edge”` |
+| `theme (optional)` | `‘system’ \| ‘dark’ \| ‘light’` | Whether to use dark or light mode. | `”system”` |
+| `featureFlags (optional)` | `FeatureFlagAdapter \| undefined` | See [Implement FeatureFlagAdapter](/product/dev-toolbar/setup//#implement-feature-flag-adapter) above | `undefined` |
```

Comment:
Why did "Implement FeatureFlagAdapter" go above?

Reply:
Jasmin and I think that it makes more sense since you need to set up the feature flags adapter before you run the init (at least if you inject). Open to moving it back down if you have a differing opinion tho!

Reply:
Flip it back. 

the way you're manually injecting it is a development thing, users won't do that in real life. and even if they did, they'd need to do it every time they opened a new browser window.

It's not a scenario that's in the docs, or that should happen often. We should organize the docs to someone can follow from steps 1 to N top to bottom. When a question pops up then it should be answered in the next section. Flag adaopters are optional, so should be lower down.